index.html

```diff
@@ -8,310 +8,339 @@
-        <div class="menu-btn__ham three"></div>
-      </nav>
-      <nav class="desktop-navigation">
-        <ul>
-          <li class="nav__items nav-mobile">
-            <a href="#portfolio">Portfolio</a>
-          </li>
-          <li class="nav__items nav-mobile"><a href="#about">About</a></li>
-          <li class="nav__items nav-mobile"><a href="#contact">Contact</a></li>
-        </ul>
-      </nav>
-    </header>
-
-    <main>
-      <section class="main headline-section">
-        <div class="headline-section__box">
-          <h1 class="headline-section__title">
-            I'm Mesfin<br />Glad to see you!
-          </h1>
-          <p class="headline-section__cont">
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contact me.
-          </p>
-          <h2 class="headline-section__sub-title">LET’S CONNECT</h2>
-          <ul class="headline-section__social">
-            <li class="headline-section__social--item">
-              <a
-                href="https://twitter.com/login?lang=en"
-                class="social-icon twitter"
-                ><img src="icons/twitter.png" alt="twitter icon"
-              /></a>
+    <header class="hamburger">
+      <div class="head">
+        <div class="name-me">Mesfin Haile</div>
+        <img id="btn" class="mob-img" src="icons/menu.png" alt="menu" />
+        <div class="desk-head">
+          <ul class="desk-li">
+            <li class="close">
+              <img id="close" src="icons/Icon.png" alt="" />
             </li>
-            <li class="headline-section__social--item">
-              <a
-                href="https://www.linkedin.com/feed/"
-                class="social-icon linkedin"
-                ><img src="icons/Linkedin.svg" alt="Linkedin icon"
-              /></a>
-            </li>
-            <li class="headline-section__social--item">
-              <a href="#" class="social-icon linkedin"
-                ><img src="icons/medium.png" alt="medium icon"
-              /></a>
-            </li>
-            <li class="headline-section__social--item">
-              <a href="https://github.com/mesfin345" class="social-icon github"
-                ><img src="icons/github.svg" alt="github icon"
-              /></a>
-            </li>
-            <li class="headline-section__social--item">
-              <a href="#" class="social-icon chill"
-                ><img src="icons/angelist.png" alt="angelist icon"
-              /></a>
+            <li class="common-li" id="portfolio">Portfolio</li>
+            <li class="common-li" id="about">About</li>
+            <li class="common-li" id="contact">Contacts</li>
+            <li class="common-li gone">
+              <img class="desk-img" alt="desktop-img" src="icons/Rec.png" />
```

Comment:
- [ ] [OPTIONAL] Let's make sure we add envelope looking icon that matches the Figma design Because it indicates that you can send mail from here instead of going to contacts.

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/155341029-36bca86a-a6e3-4234-ad64-0e97e2f1e4dd.png) | ![image](https://user-images.githubusercontent.com/87689547/155341140-e9284d03-4294-4e74-b78f-a814bda1534c.png)